interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html

```diff
@@ -151,50 +151,65 @@
 //==================================
 
```

Comment:
testing issues on this script:
1. New appointment buttons doesn't work and places following error in js log:
ReferenceError: $Date is not defined[Learn More]  index.php:1:19
onclick

2. Prior day backward arrow shows goes to white screen with following error:
Not Found
The requested URL /openemr/interface/main/calendar/{$PREV_DAY_URL} was not found on this server.

3. Next day forward arrow goes to white screen with following error.
Not Found
The requested URL /openemr/interface/main/calendar/{$NEXT_DAY_URL} was not found on this server.